changelogs/fragments/7464-fix-vm-removal-in-proxmox_pool_member.py

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - proxmox_pool_member - absent state for type vm did not delete vms from the pools (https://github.com/ansible-collections/community.general/pull/7464)
```

Comment:
First, you need to fix the filename. The extension should be `.yml`, not `.py`. Then please add a period at the end, and capitalize VM:
```suggestion
  - proxmox_pool_member - absent state for type VM did not delete VMs from the pools (https://github.com/ansible-collections/community.general/pull/7464).
```

Reply:
thank you, fixed it